databusclient/client.py

```
@@ -1,721 +1,875 @@
+            f"Can't parse Argument {metadata_list[-1]}. Too many values, submit shasum and "
+            f"content_length in the form of shasum:length"
+        )
+
+    sha256sum = last_arg_split[0]
+    content_length = int(last_arg_split[1])
+
+    return sha256sum, content_length
+
+
+def __load_file_stats(url: str) -> Tuple[str, int]:
+    resp = requests.get(url)
```

Ruff (0.14.2)

Probable use of `requests` call without timeout

(S113)